src/main/java/core/basesyntax/service/WriterImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitType;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriterImpl implements Writer {
+
+    @Override
+    public void writeRepo(Map<FruitType, Integer> fruitServiceMap) {

```

Comment:
```suggestion
    public void write(String content) {
```
```suggestion
    public void write(String[] lines) {
```
Writer shouldn't be bothered with the logic of generating report out of the map. It should be given a ready content that is meant to be written